formwidgets/BlogMarkdown.php

```diff
@@ -55,31 +55,50 @@ protected function checkUploadPostback()
         $uploadedFileName = null;
 
         try {
-            $uploadedFile = Input::file('file');
 
-            if ($uploadedFile)
-                $uploadedFileName = $uploadedFile->getClientOriginalName();
+            if (Input::has('file')) {
 
-            $validationRules = ['max:'.File::getMaxFilesize()];
-            $validationRules[] = 'mimes:jpg,jpeg,bmp,png,gif';
+                $uploadedFile = Input::file('file');
 
-            $validation = Validator::make(
-                ['file_data' => $uploadedFile],
-                ['file_data' => $validationRules]
-            );
+                if ($uploadedFile)
+                    $uploadedFileName = $uploadedFile->getClientOriginalName();
```

Comment:
@marzochi We follow PSR-2 guidelines, so all conditions need surrounding braces.
```suggestion
                if ($uploadedFile) {
                    $uploadedFileName = $uploadedFile->getClientOriginalName();
                }
```